backend/open_webui/retrieval/utils.py

```
@@ -745,38 +818,33 @@ def generate_embeddings(
+    elif engine == "azure_openai":
+        azure_api_version = kwargs.get("azure_api_version", "")
+        embeddings = generate_azure_openai_batch_embeddings(
+            model,
+            text if isinstance(text, list) else [text],
+            url,
+            key,
+            azure_api_version,
+            prefix,
+            user,
+        )
         return embeddings[0] if isinstance(text, str) else embeddings
```

Ruff (0.11.9)

Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)